### ISSUE TITLE
Installation issue with aspell and git.

### ISSUE THREAD

**Issue by SrinivasDJ** [syl20bnr/spacemacs#1259]
1. I did a git clone to install.
2. Started emacs for the first time after clone and I got the following error:
Done (Total of 2 files compiled, 1 skipped)
(Spacemacs) --> installing (spacemacs):zenburn-theme... [88/88]
Contacting host: melpa.org:80 [2 times]
Generating autoloads for zenburn-theme.el...done
Saving file /home/user/.emacs.d/elpa/zenburn-theme-20150315.1540/zenburn-theme-autoloads.el...
Wrote /home/user/.emacs.d/elpa/zenburn-theme-20150315.1540/zenburn-theme-autoloads.el
(No changes need to be saved)
(No files need saving)
Checking /home/user/.emacs.d/elpa/zenburn-theme-20150315.1540... [2 times]
Showing all blocks ... done
Compiling /home/user/.emacs.d/elpa/zenburn-theme-20150315.1540/zenburn-theme-pkg.el...done
Wrote /home/user/.emacs.d/elpa/zenburn-theme-20150315.1540/zenburn-theme-pkg.elc
Checking /home/user/.emacs.d/elpa/zenburn-theme-20150315.1540... [2 times]
Done (Total of 1 file compiled, 2 skipped)
Starting new Ispell process [aspell::english] ...
Error enabling Flyspell mode:
(Searching for program no such file or directory aspell)
Helm completion enabled
Loading /home/user/.emacs.d/spacemacs/keybindings.el (source)...done
Start checking for new version...
Starting new Ispell process [aspell::english] ...
Error enabling Flyspell mode:
(Searching for program no such file or directory aspell)
Spacemacs is ready.
error in process sentinel: if: Searching for program: no such file or directory, git
error in process sentinel: Searching for program: no such file or directory, git

3. Closed the emacs and started once more. Got the same error. I didn't get any message asking for vi or emacs mode after installation. Is it because of the error? How can I get rid of these errors which I get everytime I start emacs? I have aspell in my path.

Loading 00debian-vars...done
Loading /etc/emacs/site-start.d/50autoconf.el (source)...done
Loading /etc/emacs/site-start.d/50dictionaries-common.el (source)...
Loading debian-ispell...
Loading /var/cache/dictionaries-common/emacsen-ispell-default.el (source)...done
Loading debian-ispell...done
Loading /var/cache/dictionaries-common/emacsen-ispell-dicts.el (source)...done
Loading /etc/emacs/site-start.d/50dictionaries-common.el (source)...done
Loading /etc/emacs/site-start.d/50silversearcher-ag-el.el (source)...done
Loading /etc/emacs/site-start.d/50sml-mode.el (source)...
Loading sml-mode-startup...done
Loading /etc/emacs/site-start.d/50sml-mode.el (source)...done
Loading /home/user/.emacs.d/core/core-load-paths.el (source)...done
Loading /home/user/.spacemacs...done
(Spacemacs) Warning: Cannot find font "Source Code Pro"!
Mark set [2 times]
Loading /home/user/.emacs.d/.cache/spacemacs-buffer.el (source)...done
Mark set
Saving file /home/user/.emacs.d/.cache/spacemacs-buffer.el...
Wrote /home/user/.emacs.d/.cache/spacemacs-buffer.el
Loading /home/user/.emacs.d/spacemacs/funcs.el (source)...done
Loading /home/user/.emacs.d/spacemacs/config.el (source)...done
Loading /home/user/.emacs.d/spacemacs/packages.el (source)...done
Loading /home/user/.emacs.d/spacemacs/extensions.el (source)...done
Loading /home/user/.emacs.d/spacemacs/keybindings.el (source)...done
Loading /home/user/.emacs.d/.cache/recentf...done
Start checking for new version...
ad-handle-definition: `find-function-search-for-symbol' got redefined
Starting new Ispell process [aspell::english] ...
Error enabling Flyspell mode:
(Searching for program no such file or directory aspell)
Starting new Ispell process [aspell::english] ...
Error enabling Flyspell mode:
(Searching for program no such file or directory aspell)
Spacemacs is ready.
error in process sentinel: async-handle-result: Searching for program: no such file or directory, git
error in process sentinel: Searching for program: no such file or directory, git

--
Thanks.

**Comment by syl20bnr** [comment]
Indeed, we have to fix, this.
Related issue #9 (oh yeah pretty old one :-)).

**Comment by Poorchop** [comment]
I'm not sure if the missing git binary is also an issue in Linux, but in Windows, I'm pretty positive that I'm getting `error in process sentinel: Searching for program: no such file or directory, git` because I installed git via GitHub's installer, and it's a non-standard location. I checked in ~/.spacemacs and I couldn't find any place to specify the location of git.exe. I also went through the readme for the git contribution layer and I didn't see any instructions as to how I could point to the correct location of git.exe.
